client/src/components/Menu/index.tsx

```diff
@@ -118,8 +121,9 @@ class Menu extends React.Component<IMenuProps> {
     }
 
     public render(): JSX.Element {
-        const {loggedUsername, newUsername, searchResults, loaded}: IMenuProps = this.props;
+        const {loggedUsername, newUsername, searchResults, loaded, loading}: IMenuProps = this.props;
         const {searchValue}: IMenuState = this.state;
+        const RESULTS_PER_PAGE = 5;
```

Comment:
Pls move to config and reuse

Reply:
I actually don't use it here anymore, so I removed it.